cds/modules/fixtures/data/pages/faq.html

```diff
@@ -42,10 +42,28 @@ <h2 id="cds-videos">CDS Videos</h2>
 <p>The documents on the CERN Document Server are generally already the highest quality available.</p>
 <p><strong>What about low resolution multimedia files?</strong></p>
 <p>Low-res files are also protected by the CERN copyright, and subject to the same conditions of use as high resolutions files. Though you will not need to login to access them, you agree to the same conditions (including the CERN computing rules) by downloading this material.</p>
+<p><strong><a name="embed" class="cds-anchor">Do I have special options of embeding a video?</a></strong></p>
+<p>Yes, you can add an URL parameter to the link in your iframe if you need to specify embed options. Below you can find a list of them:</p>
+<dl>
+  <dt>autoplay=1</dt><dd>this parameter triggers auto playing video when the page is loaded. Please note that some web browsers turn off the sound for autplayed videos.</dd>
+  <dt>controlsOff=1</dt><dd>this parameter hides the seeking bar, video tools bar and video title</dd>
+  <dt>muted=1</dt><dd>this parameter turns off the sound of the video</dd>
+  <dt>start=5</dt><dd>this parameter indicates that the video should start playing at 5th second</dd>
+  <dt>stop=30</dt><dd>this parameter indicates that the video should stop playing at 30th second</dd>
+  <dt>subtitlesOff=1</dt><dd>this parameter turns off the subtitles of the video (if the video has them)</dd>
+  <dt>loop=1</dt><dd>this parameter indicates that the video should play in loop</dd>
+  <dt>responsive=1</dt><dd>this parameter indicates that video should fill out the container, and to be responsive</dd>
+</dl>
+<p>Please find the example of iframe with added special parameters below (this case the video will play in loop and it will be muted):</p>
```

Comment:
Minor: maybe remove the Please?
Here an example of how to embed a video muted and that will play in loop

or similar